README.md

```diff
@@ -1,5 +1,9 @@
 # TrackGit
 
+![Screenshot1](screenshot_trackgit1.png)
+
+![Screenshot2](screenshot_trackgit.png)
+
```

Comment:
Can you put th two screenshots side by side?
Also, the first shot has some weird garbage at the bottom right corner. The URL field could contain TrackGit's repo url to clone.
The second shot should contain the mouse pointer.

Reply:
That artifact isn't there with the TrackGit that I installed from HaikuDepot (32bit). You can use this: https://0x0.st/sI4P.png

The mouse pointer is only needed when showing an open menu.